app/Post_Types/Diviner_Field/Diviner_Field.php

```diff
@@ -33,7 +33,7 @@ public function register() {
 		register_post_type( static::NAME, $args );
 	}
 
-	public function get_active_fields($additional_meta_query = []) {
+	static public function get_active_fields($additional_meta_query = []) {
```

Comment:
should be `public static`, not `static public`; not a real issue but one that will be flagged by any coding standard.